### PR TITLE
[SDK-1578] Scene tree init fix

### PR DESCRIPTION
**Pull request by danschultz**
## What

Should address a bug where the scene tree would not get initialized. This can happen if the viewer has already been loaded.

## Ticket

https://vertexvis.atlassian.net/browse/SDK-1578

### CODE REVIEW

File: packages/viewer/src/components/scene-tree/scene-tree.tsx [hunk 1]
```diff
@@ -50,6 +50,9 @@ export class SceneTree {
   @Prop({ reflect: true, mutable: true })
   public controller: SceneTreeController | undefined;
 
+  @Prop({ reflect: true, mutable: true })
+  public client!: SceneTreeAPIClient;
```
Comment (jdm717) [inline packages/viewer/src/components/scene-tree/scene-tree.tsx:54]: Do we want this API client to be `reflect`ed here? Not sure it makes sense as a DOM attribute, unless I'm missing something here?

File: packages/viewer/src/components/scene-tree/scene-tree.tsx [hunk 2]
```diff
@@ -238,70 +241,85 @@ export class SceneTree {
...
+      );
 
-      if (newViewer != null) {
-        newViewer.addEventListener('sceneReady', this.handleViewerSceneReady);
-        newViewer.addEventListener(
-          'connectionChange',
-          this.handleViewerConnectionStatusChange
-        );
+      const isSceneReady = await newViewer.isSceneReady();
+      console.log('is scene ready', isSceneReady);
```
Comment (jdm717) [inline packages/viewer/src/components/scene-tree/scene-tree.tsx:266]: Can we move this to `console.debug` or remove?

**Review by jdm717 — APPROVED** [review]
LGTM aside from a couple nits

File: packages/viewer/src/components/scene-tree/scene-tree.tsx (hunk 2 above)
Comment (danschultz) [reply packages/viewer/src/components/scene-tree/scene-tree.tsx:266]: Will remove.

File: packages/viewer/src/components/scene-tree/scene-tree.tsx (hunk 1 above)
Comment (danschultz) [reply packages/viewer/src/components/scene-tree/scene-tree.tsx:54]: I have a follow up PR to address this.

**Pull request merged**